products/bazel.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Bazel
+category: app
+tags: google build-tool java-runtime
+iconSlug: bazel
```

Comment:
We use https://simpleicons.org/ for the icons and bazel is not listed yet. Feel free to reach out to the devs at https://github.com/simple-icons/simple-icons. Meanwhile remove the icon:
```suggestion
```

Reply:
good to know, let me add some efforts into simple-icons with this.

Reply:
- relates to https://github.com/simple-icons/simple-icons/issues/8166